Engine/Helper.cs

```diff
@@ -3555,4 +3555,239 @@ public object VisitUsingExpression(UsingExpressionAst usingExpressionAst)
             return null;
         }
     }
+
+    /// Class to represent a directed graph
+    public class Digraph<T>
```

Comment:
We use Pester for our testing purposes. If I make it internal, how do I go about testing? I do not think I will be able find the type in Pester tests if it is internal.